src/main/java/com/microsoft/spring/data/gremlin/conversion/script/GremlinScriptLiteralGraph.java

```diff
@@ -76,4 +76,19 @@
 
         return this.generateDeleteAllScript(source);
     }
+
+    public List<String> generateIsEmptyScript(@NonNull GremlinSource source) {
+        if (!(source instanceof GremlinSourceGraph)) {
+            throw new GremlinUnexpectedSourceTypeException("should be the instance of GremlinSourceGraph");
+        }
+
+        final List<String> scriptList = new ArrayList<>();
+
+        scriptList.add(Constants.GREMLIN_PRIMITIVE_GRAPH);
+        scriptList.add(Constants.GREMLIN_PRIMITIVE_VERTEX_ALL);
+
+        final String query = String.join(Constants.GREMLIN_PRIMITIVE_INVOKE, scriptList);
+
+        return Arrays.asList(query);
```

Comment:
Based on this implementation, it seems list only have one element. If there's no other implementation return many elements, we could make return type as String.

Reply:
As it will be executed by outer executor, I keep List for voiding extra if condition for executor.